public/app/features/dashboard/components/DashboardLoading/DashboardLoading.tsx

```diff
@@ -34,12 +34,22 @@ export const DashboardLoading = ({ initPhase }: Props) => {
 };
 
 export const getStyles = (theme: GrafanaTheme) => {
+  // Amount of time we want to pass before we start showing loading spinner
+  const slowStartThreshold = '0.5s';
+
+  const delay = keyframes`
```

Comment:
minor nit - `delay` is a bit confusing since this doesn't really have anything to do with the delay. it's just the animation to show/hide.